completions/bash/brew

```diff
@@ -563,10 +563,10 @@ _brew() {
 
   if [[ "$i" -eq "$COMP_CWORD" ]]
   then
-    # Do not auto-complete "instal" abbreviation for "install" command.
+    # Do not auto-complete "*instal" abbreviations for "*install" commands.
```

Comment:
`# Do not auto-complete "instal" or "uninstal" abbreviations for "*install" commands.` I think is a little easier to parse.